lib/registrant.rb

```diff
@@ -0,0 +1,18 @@
+class Registrant 
+  attr_reader :name, :age, :license_data 
+  def initialize(name, age, permit = false ) 
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
+  end
+
+  def permit?
+    @permit
+  end
+
+  def earn_permit 
+    @permit = true
```

Comment:
You could also add what is called a guard clause here that would check
```ruby
if @age >= 15
```
That we they could only `earn_permit` if they were of the correct age to do so.